packages/contracts-bedrock/deploy-config/mainnet.json

```diff
@@ -0,0 +1,34 @@
+{
```

Comment:
Can we name this something like `mainnet-forked`?

Reply:
Wouldn't we need to duplicate the deployment to do that? Hardhat deploy config expects the names to all match.